### PR TITLE
En_Kendo_Js

### PR DESCRIPTION
**Pull request by Kelebek1**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

File: src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c [hunk 1]
```diff
@@ -28,86 +49,720 @@ const ActorInit En_Kendo_Js_InitVars = {
...
+                    break;
+
+                case 2:
+                    phi_v0 = 2;
+                    break;
+
+                default:
+                    phi_v0 = 0;
+                    break;
+            }
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c:186]:
```suggestion
            switch (gSaveContext.playerForm) {
                case PLAYER_FORM_DEKU:
                    phi_v0 = 0;
                    break;

                case PLAYER_FORM_GORON:
                    phi_v0 = 1;
                    break;

                case PLAYER_FORM_ZORA:
                    phi_v0 = 2;
                    break;

                default:
                    phi_v0 = 0;
                    break;
            }
```

File: src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c [hunk 2]
```diff
@@ -28,86 +49,720 @@ const ActorInit En_Kendo_Js_InitVars = {
...
+
+            func_801518B0(globalCtx, D_80B27CE0[phi_v0][sp30], &this->actor);
+            this->unk_288 = D_80B27CE0[phi_v0][sp30];
+        } else {
+            if ((Player_GetMask(globalCtx) != PLAYER_MASK_NONE) && (Player_GetMask(globalCtx) < PLAYER_MASK_GIANT)) {
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c:191]: This `else if` can be merged

File: src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c [hunk 3]
```diff
@@ -28,86 +49,720 @@ const ActorInit En_Kendo_Js_InitVars = {
...
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 36.0f);
+
+    Collider_InitCylinder(globalCtx, &this->collider);
+    Collider_SetCylinder(globalCtx, &this->collider, &this->actor, &sCylinderInit);
+    CollisionCheck_SetInfo2(&this->actor.colChkInfo, DamageTable_Get(0x16), &sColChkInfoInit);
+
+    SkelAnime_InitFlex(globalCtx, &this->skelAnime, &object_js_Skel_006990, &object_js_Anim_000F4C, this->jointTable,
+                       this->morphTable, 0xD);
+
+    if ((CURRENT_DAY == 3) && (!(gSaveContext.time < 0xF556) || (gSaveContext.time < CLOCK_TIME(6, 0)))) {
```
Comment (hensldm) [inline src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c:112]:
```suggestion
    if ((CURRENT_DAY == 3) && !((gSaveContext.time <= CLOCK_TIME(23, 0)) && (gSaveContext.time >= CLOCK_TIME(6, 0)))) {
```
Should still match, and looks cleaner. I prefer the ! with the &&, but could be demorgans to just a || if desired.

**Review by hensldm — APPROVED** [review]
Just one if statement cleanup suggestion.

File: src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c (hunk 3 above)
Comment (Kelebek1) [reply src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c:112]: Time off off-by-one and the condition can't be changed like this.

**Review by AngheloAlf — APPROVED** [review]

File: src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c (hunk 3 above)
Comment (hensldm) [reply src/overlays/actors/ovl_En_Kendo_Js/z_en_kendo_js.c]: Hmm, I've seen Ido be fine with these kind of demorgans before. I tried checking out your branch, and my suggestion seems fine. I still get matching.

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**